src/RouterCLILookupCodegenBuilder.hack

```diff
@@ -201,6 +209,7 @@ final class RouterCLILookupCodegenBuilder {
       ->addLine('exit(1);')
       ->endIfBlock()
       ->addLine('require_once($autoloader);')
+      ->addLine('\Facebook\AutoloadMap\initialize();')
```

Comment:
double backslashes?

Reply:
Sure...I tend to omit them in single quotes, since they only support \\ and \' so there's not much chance of an accidental escape sequence, but I guess it's better to be consistent.